frontend/src/app/components/wp-new/wp-create.component.ts

```diff
@@ -118,7 +118,7 @@ export class WorkPackageCreateComponent extends UntilDestroyedMixin implements O
       .createdWorkPackage()
       .then((changeset:WorkPackageChangeset) => {
         this.change = changeset;
-        this.newWorkPackage = changeset.projectedResource;
+        this.newWorkPackage = changeset.pristineResource;
```

Comment:
If we're not using `pristineResource` here, we're passing a proxied work package into the HalResourceEditingService below